packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerService.ts

```diff
@@ -3,97 +3,90 @@ import type {
   OpenId4VciCreateCredentialOfferOptions,
   OpenId4VciCreateIssuerOptions,
   OpenId4VciPreAuthorizedCodeFlowConfig,
-  OpenId4VcIssuerMetadata,
-  OpenId4VciSignSdJwtCredential,
-  OpenId4VciSignW3cCredential,
-  OpenId4VciSignMdocCredential,
+  OpenId4VciSignW3cCredentials,
```

Comment:
Same comment as with the holder service file. Would really benefit from a split up in files and some helper functions to make it a lot easier to read.